<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNovyUzivatel</class>
 <widget class="QDialog" name="DialogNovyUzivatel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Založení uživatele</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Typ uživatele:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Heslo:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Potvrzení hesla:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Celé jméno:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Uživatelské jméno:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QRadioButton" name="volbaBezny">
       <property name="text">
        <string>běžný uživatel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QRadioButton" name="volbaAdmin">
       <property name="text">
        <string>administrátor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="vstupUzivJmeno">
       <property name="toolTip">
        <string>Uživatelské jméno</string>
       </property>
       <property name="placeholderText">
        <string>jako do Farmisu</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="vstupCeleJmeno">
       <property name="toolTip">
        <string>Celé jméno</string>
       </property>
       <property name="placeholderText">
        <string>tituly, jméno, příjmení, tituly</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="vstupHeslo">
       <property name="toolTip">
        <string>Heslo</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="vstupHeslo2">
       <property name="toolTip">
        <string>Potvrzení hesla</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="popisekUpozorneni">
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="tlacitkoZrusit">
       <property name="toolTip">
        <string>Zrušit (Esc)</string>
       </property>
       <property name="text">
        <string>Zrušit</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tlacitkoZalozit">
       <property name="toolTip">
        <string>Uložit (Ctrl+Enter)</string>
       </property>
       <property name="text">
        <string>Založit</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>vstupUzivJmeno</tabstop>
  <tabstop>vstupCeleJmeno</tabstop>
  <tabstop>vstupHeslo</tabstop>
  <tabstop>vstupHeslo2</tabstop>
  <tabstop>volbaBezny</tabstop>
  <tabstop>volbaAdmin</tabstop>
  <tabstop>tlacitkoZrusit</tabstop>
  <tabstop>tlacitkoZalozit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
